components/ExhibitionsComponents/AllExhibitions/components/ExhibitionsList/ExhibitionsList.module.css

```
@@ -82,49 +80,49 @@ $imageMaxWidth: 10rem;
 
-    @media (min-width: $largeRem) {
+    @media (min-width: 72em) {
       font-size: 1.275rem;
     }
   }
 }
 
 .exhibitionList {
-  display: flex; // for ie11
-  flex-direction: column; // for ie11
+  display: flex; 
```

Stylelint (17.13.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)